trio/_core/_run.py

```diff
@@ -1,43 +1,37 @@
+import functools
 import inspect
-import enum
-from collections import deque
-import threading
-from time import monotonic
+import logging
 import os
 import random
-from contextlib import contextmanager, closing
 import select
-import sys
+import threading
+from collections import deque
+from contextlib import contextmanager, closing
+from contextvars import copy_context
 from math import inf
-import functools
-import logging
+from time import monotonic
 
 import attr
-from sortedcontainers import SortedDict
 from async_generator import async_generator, yield_
+from sortedcontainers import SortedDict
 
-from .._util import acontextmanager
-from .._deprecate import deprecated
-
-from .. import _core
-from ._exceptions import (
-    TrioInternalError, RunFinishedError, Cancelled, WouldBlock
+from . import _public
+from ._entry_queue import EntryQueue, TrioToken
+from ._exceptions import (TrioInternalError, RunFinishedError, Cancelled)
+from ._ki import (
+    LOCALS_KEY_KI_PROTECTION_ENABLED, currently_ki_protected, ki_manager,
+    enable_ki_protection
 )
 from ._multierror import MultiError
 from ._result import Result, Error, Value
 from ._traps import (
-    cancel_shielded_checkpoint,
     Abort,
     wait_task_rescheduled,
     CancelShieldedCheckpoint,
     WaitTaskRescheduled,
 )
-from ._entry_queue import EntryQueue, TrioToken
-from ._ki import (
-    LOCALS_KEY_KI_PROTECTION_ENABLED, currently_ki_protected, ki_manager,
-    enable_ki_protection
-)
-from . import _public
+from .. import _core
+from .._util import acontextmanager
```

Comment:
I guess your editor is set up to automatically sort imports? I'm not going to stress over it here, but it does make it hard to tell what's changed... do you think you could get it to not do this in future PRs? (Or alternatively, if you absolutely can't stand unsorted imports, I guess you could set up [`isort`](https://github.com/timothycrosley/isort) alongside `yapf` so everyone does it? :-))

Reply:
Oh, this was an accident; I have PyCharm set to auto-optimize imports by default on my global scope which removes unused ones and re-arranges them into a different order. I forgot to turn it off for this project.